### ISSUE TITLE
Print zbctl status as json

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#5864]
**Is your feature request related to a problem? Please describe.**
Currently we print the topology of the cluster via `zbctl status` and it looks like this:

```
[zell scripts/ ns:zell-chaos]$ k exec -it zell-chaos-zeebe-gateway-5577d6958-r6bfp -- zbctl status --insecure
Cluster size: 3
Partitions count: 3
Replication factor: 3
Gateway version: 0.26.0-SNAPSHOT
Brokers:
  Broker 0 - zell-chaos-zeebe-0.zell-chaos-zeebe.zell-chaos.svc.cluster.local:26501
    Version: 0.26.0-SNAPSHOT
    Partition 1 : Follower, Healthy
    Partition 2 : Follower, Healthy
    Partition 3 : Follower, Healthy
  Broker 1 - zell-chaos-zeebe-1.zell-chaos-zeebe.zell-chaos.svc.cluster.local:26501
    Version: 0.26.0-SNAPSHOT
    Partition 1 : Follower, Healthy
    Partition 2 : Follower, Healthy
    Partition 3 : Follower, Healthy
  Broker 2 - zell-chaos-zeebe-2.zell-chaos-zeebe.zell-chaos.svc.cluster.local:26501
    Version: 0.26.0-SNAPSHOT
    Partition 1 : Leader, Healthy
    Partition 2 : Leader, Healthy
    Partition 3 : Leader, Healthy
```

This is totally fine for a human beeing, but not if you what to process it.

**Describe the solution you'd like**

It would be cool if we had a flag or something where we can tell `zbctl` to print the topology as json.

**Describe alternatives you've considered**
idk - self parsing

**Additional context**

I need this quite often in building new chaos experiments where I want to know who is the leader for a certain partition or who is taking part of the partition. For our benchmarks it currently works kind of if the partitions are well distributed, like three nodes and three partitions, then you can do something like:

```sh
state=$1 # example Leader
partition=${2:-3}

# For cluster size 3 and replication factor 3
# we know the following partition matrix
# partition \ node  0    1     2
#     1             L    F     F
#     2             F    L     F
#     3             F    F     L
#    etc.
# This means broker 1, 2 or 3 participates on partition 3
# BE AWARE the topology above is just an example and the leader can every node participating node.


index=$(($(echo "$topology" \
  | grep "Partition $partition" \
  | grep -n "$state" -m 1 \
  | sed 's/\([0-9]*\).*/\1/') - 1))

pod=$(echo "$pod" | sed 's/\(.*\)\([0-9]\)$/\1/')
pod="$pod$index"

```

but this doesn't work if you have not well distributed partitions like 5 nodes and 3 partition, or 5 nodes 8 partition and replication 3 etc.

**Comment by Zelldon** [comment]
blocks https://github.com/zeebe-io/zeebe-cluster-testbench/issues/154

**Comment by aivinog1** [comment]
@korthout Hi! If you are not already done with this issue, I can take it :)

**Comment by korthout** [comment]
Hi @aivinog1. I thought I could resolve this quickly, but ran into an issue. Feel free to take over.

I added a testcase to `clients/go/cmd/zbctl/main_test.go` with a new golden file, and everything seemed to go smoothly until I noticed that the enum values (both partition role and health) are not converted to json properly using the current marshaller. This has to do with the protobuf files. I recommend you try the same, run into the issue and then take a look at 
`github.com/golang/protobuf/jsonpb` and `google.golang.org/protobuf/runtime/protoiface` to replace the `printJSON` function implementation in `clients/go/cmd/zbctl/internal/commands/root.go`.

Let me know if you have any questions and good luck 🍀
